test/src/e2e.long/mempoolMinfee.test.ts

```diff
@@ -0,0 +1,127 @@
+import "mocha";
+import {
+    validator0Address,
+    validator1Address,
+    validator2Address
+} from "../helper/constants";
+import CodeChain from "../helper/spawn";
+
+chai.use(chaiAsPromised);
+
+describe("MemPoolMinFees", async function() {
+    const chain = `${__dirname}/../scheme/tendermint-int.json`;
+    const config1 = `test/src/config/mem-pool-min-fee1.toml`;
+    const config2 = `test/src/config/mem-pool-min-fee2.toml`;
+    const node1_with_min_pay_fee_150 = new CodeChain({
+        chain,
+        argv: [
+            "--engine-signer",
+            validator0Address.toString(),
+            "--password-path",
+            "test/tendermint/password.json",
+            "--force-sealing",
+            "--config",
+            config1
+        ],
+        additionalKeysPath: "tendermint/keys"
+    });
+    const node2_with_min_pay_fee_200 = new CodeChain({
+        chain,
+        argv: [
+            "--engine-signer",
+            validator1Address.toString(),
+            "--password-path",
+            "test/tendermint/password.json",
+            "--force-sealing",
+            "--config",
+            config2
+        ],
+        additionalKeysPath: "tendermint/keys"
+    });
+    const node3 = new CodeChain({
+        chain,
+        argv: [
+            "--engine-signer",
+            validator2Address.toString(),
+            "--password-path",
+            "test/tendermint/password.json",
+            "--force-sealing"
+        ],
+        additionalKeysPath: "tendermint/keys"
+    });
+
+    beforeEach(async function() {
+        await node1_with_min_pay_fee_150.start();
+        await node2_with_min_pay_fee_200.start();
+        await node3.start();
+
+        await node1_with_min_pay_fee_150.connect(node2_with_min_pay_fee_200);
+        await node1_with_min_pay_fee_150.connect(node3);
+        await node2_with_min_pay_fee_200.connect(node3);
+
+        await node1_with_min_pay_fee_150.waitPeers(2);
+        await node2_with_min_pay_fee_200.waitPeers(2);
+        await node3.waitPeers(2);
+    });
+
+    it("two nodes with different minimum pay costs1", async function() {
+        const tx = await node1_with_min_pay_fee_150.sendPayTx({
+            fee: 175,
+            quantity: 100_000,
+            recipient: validator0Address
+        });
+        await node1_with_min_pay_fee_150.waitBlockNumber(2);
+        await node2_with_min_pay_fee_200.waitBlockNumber(2);
+        await node3.waitBlockNumber(2);
+
+        expect(
+            await node1_with_min_pay_fee_150.sdk.rpc.chain.containsTransaction(
+                tx.hash()
+            )
+        ).to.be.true;
+        expect(
+            await node2_with_min_pay_fee_200.sdk.rpc.chain.containsTransaction(
+                tx.hash()
+            )
+        ).to.be.true;
+        expect(await node3.sdk.rpc.chain.containsTransaction(tx.hash())).to.be
+            .true;
+    });
+
+    it("two nodes with different minimum pay costs2", async function() {
```

Comment:
There is actually three nodes. The title is not descriptive. You should change the title to "should block the tx whose fee is less than min_fee" or some similar and concise one.

Reply:
Changed the title to a descriptive one.